notNeededPackages.json

```diff
@@ -5775,6 +5775,10 @@
             "libraryName": "styled-components",
             "asOfVersion": "6.0.0"
         },
+        "styled-components-react-native": {
+            "libraryName": "styled-components/native",
```

Comment:
I don't think this is the name of the library. This should be an npm package name.